src/com/example/TicTacToe.java

```diff
@@ -1,9 +1,46 @@
 package com.example;
 
 public class TicTacToe {
+    private static int cx,co;
```

Comment:
The count function should just return these values instead of having class global variables and setting them